apps/src/sites/studio/pages/levels/_teacher_panel.js

```diff
@@ -52,8 +47,10 @@ function renderTeacherContentToggle(store) {
   const isBlocklyOrDroplet = !!(window.appOptions && appOptions.app);
 
   ReactDOM.render(
-    <Provider store={getStore()}>
-      <TeacherContentToggle isBlocklyOrDroplet={isBlocklyOrDroplet} />
+    <Provider store={store}>
+      <TeachersOnly>
```

Comment:
Previously `renderTeacherContentToggle` would only get called when teacher_panel.html.haml partial existed, now teacher_panel.html.haml partial will always exist (because on the server we won't know yet if it's a teacher), so we need to check on the client that it will only render for teachers. I'm not too sure what this component is for, but I know it's only meant for teachers